core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -44,6 +45,8 @@
 
     private static final Pattern VALID_PATTERN = Pattern.compile("(/[^/{}:]+|/:[^/{}]+|/\\{[^/{}]+})+/?");
 
+    private static final Pattern CAPTURE_THE_REST_PATTERN = Pattern.compile("/\\{\\*([^/{}]*)}");
```

Comment:
`{**}` and `{*}` looks invalid patterns. How about excluding those patterns together?

Reply:
I see. I added the validation for the variable name. The variable name means string between `{*` and `}`.
It allows only using alphabet, numbers, and underscores, just like common variable names in java.